docs/european-user-consent.mdx

```diff
@@ -83,6 +93,13 @@ The method returns an `AdsConsentInfo` interface, which provides information abo
   - `OBTAINED`: User consent already obtained.
 - **isConsentFormAvailable**: A boolean value. If `true` a consent form is available.
 
+**Note:** The status returned by this call tells whether the user has already
+seen and replied to the consent form, rather than the actual status of his consent
```

Comment:
```suggestion
**Note:** The return status of this call is the status of *form presentation and response collection*, **not
the the actual user consent**. It simply indicates if you now have a consent response to decode.
```

That is really subtle, I can see how that was confusing and I remember the discussion around the same.
Everyone likes to propose wording changes and I'm no different :laughing: - your message definitely clarified
it for me, and with this change I try to make it even more clear, what do you think?

If you don't like it that's fine, I'll go +1 on your version, it's a big help just that it exists and I don't want to bikeshed more than one round

Reply:
Looks good to me.